### ISSUE TITLE
Fedora 40 Killswitch Problem

### ISSUE THREAD

**Issue by onurbbr** [ProtonVPN/proton-vpn-gtk-app#52]
>  We are happy to answer your questions about the code or discuss technical ideas. 
 
Please complete the following checklist (by adding [x]):
- [x] I have searched open and closed issues for duplicates
- [x] This isn't a feature request
- [ ] This is not a report about my app not working as expected

----------------------------------------

Distro: Fedora Linux 40
DE: KDE Plasma 6
Kernel: 6.8.9-300.fc40.x86_64

When I start proton-vpn-gtk-app from the terminal, I get an error about killswitch. The output is as follows:

```bash
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/proton/vpn/killswitch/interface/killswitch.py", line 57, in get
    return Loader.get("killswitch", class_name=class_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/loader/loader.py", line 113, in get
    raise RuntimeError(f"Loader: couldn't find an acceptable implementation for {type_name}.")
RuntimeError: Loader: couldn't find an acceptable implementation for killswitch.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/bin/protonvpn-app", line 33, in <module>
    sys.exit(load_entry_point('proton-vpn-gtk-app==4.3.1', 'console_scripts', 'protonvpn-app')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/vpn/app/gtk/__main__.py", line 34, in main
    controller = Controller.get(executor)
                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/vpn/app/gtk/controller.py", line 59, in get
    executor.submit(controller.initialize_vpn_connector).result()
  File "/usr/lib64/python3.12/concurrent/futures/_base.py", line 456, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/usr/lib/python3.12/site-packages/proton/vpn/app/gtk/controller.py", line 99, in initialize_vpn_connector
    self._connector = await self._api.get_vpn_connector()
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/vpn/core/api.py", line 56, in get_vpn_connector
    vpn_connector = await VPNConnector.get_instance(settings)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/vpn/connection/vpnconnector.py", line 64, in get_instance
    cls._instance = VPNConnector(settings, kill_switch=kill_switch)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/vpn/connection/vpnconnector.py", line 86, in __init__
    self._kill_switch = kill_switch or KillSwitch.get()()
                                       ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/proton/vpn/killswitch/interface/killswitch.py", line 59, in get
    raise MissingKillSwitchBackendDetails(excp) from excp
proton.vpn.killswitch.interface.exceptions.MissingKillSwitchBackendDetails: Loader: couldn't find an acceptable implementation for killswitch.
```

**Comment by calexandru2018** [comment]
I assume you've installed only the `proton-vpn-gtk-app` and not the `proton-vpn-gnome-desktop` meta-package, as installing the meta-package brings in all dependencies and I'm myself running on F40 and it's working well (even with the experimental WireGuard package).

**Comment by thgoebel** [comment]
I also ran into this. I installed `proton-vpn-gtk-app` because I am on KDE Plasma and thought that  `proton-vpn-gtk-app`  would be the general app (that runs on anything with GTK) and that `proton-vpn-gnome-desktop` has GNOME-specific additions.

Should I have followed the guide instead of making assumptions? Yes. Should Proton choose a better naming scheme that doesn't lead to people making the wrong assumptions? Also yes.
